### ISSUE TITLE
Add automated UI testing with playwright and axe dev tools

### ISSUE THREAD

**Issue by Norby257** [Norby257/SimplySupplements#20]
Implement the following tests: 

1.  Playwright tests to automate browser interactions and verify e expected behavior 
2. use axe dev tools for accessibility auditing.
3. Add both into a GitHub action, since this is a useful step to have automated a11y checks in a CI

DETAILSS 
* Run axe audit on key pages 
*  Use playwright for E2E flows 
* run this action on pull requests 
* fail build on WCAG violations.

**Comment by Norby257** [comment]
Closed because this has been merged.
